infra/modules/app/ecs.tf

```diff
@@ -46,13 +51,53 @@ resource "aws_ecs_task_definition" "cheerful" {
       {
         "name": "DATABASE_URL",
         "value": "postgresql://${aws_db_instance.app.username}:${aws_db_instance.app.password}@${aws_db_instance.app.address}:${aws_db_instance.app.port}/${aws_db_instance.app.db_name}?schema=public"
+      },
+      {
+        "name": "JWT_SIGNING_KEY",
```

Comment:
**issue:** missing DOMAIN here

Reply:
```
     {
        "name": "DOMAIN",
        "value": "dev.getcheerful.com"
      }
```

Can the DOMAIN be just hardcoded like this?

Reply:
It is available as `local.environment_domain`

Reply:
Cool!